src/vma/util/vma_stats.h

```diff
@@ -358,9 +398,4 @@ void print_full_stats(socket_stats_t* p_si_stats, mc_grp_info_t* p_mc_grp_info,
 void print_netstat_like(socket_stats_t* p_si_stats, mc_grp_info_t* p_mc_grp_info, FILE* file, int pid);
 void print_netstat_like_headers(FILE* file);
 
```

Comment:
You can also remove the` <bitset>` includes from :
stats_reader.cpp : line 44
stats_printer.cpp : line 44